src/main/java/javax/validation/ConstraintValidatorContext.java

```diff
@@ -567,9 +567,9 @@ ContainerElementNodeBuilderCustomizableContext addContainerElementNode(
 
 		/**
 		 * Represents refinement choices for a node which is
-		 * in an {@code Iterator} or {@code Map}.
+		 * in an iterable, e.g. array, {@code Iterable} or {@code Map}.
 		 * <p>
-		 * If the iterator is an indexed collection or a map,
+		 * If the iterable is an indexed collection or a map,
```

Comment:
This type here is returned by `inIterable()` and could apply to a "non-container" in our sense, i.e. an array. Whereas the mention below applies to containers only.